app/main.py

```diff
@@ -26,30 +26,49 @@ def load_data(file_path: str) -> list[dict]:
     return data
 
 
-def video_with_highest_views(videos: list[dict]) -> str:
-    pass
+def video_with_highest_views(videos) -> str:
+    return max(videos, key=lambda x: x.get("views", 0)).get("title")
 
 
-# ration = likes / views for 1 video!
-# average ratio it's sum of all rations / count of videos
 def average_likes_to_views_ratio(videos: list[dict]) -> float:
-    pass
+    viewed_movies = [movie for movie in videos if movie.get("views") != 0]
+
+    return sum(movie.get("likes") / movie.get("views") for movie in viewed_movies) / len(viewed_movies)
 
```

Comment:
You can simplify your code for only 1 iteration
`[movie.get("likes") / movie.get("views" for movie in videos if movie.get("views") != 0]`
and then in sum use only variable and for len it will be the same value

Reply:
Yep, I fully agree. Unfortunately I even couldn't think there is such cool realisation into get method